fulp_modules/lisa/joel_gun.dm

```diff
@@ -0,0 +1,195 @@
+		to_chat(user, "<span class='warning'>You don't want to waste [src]'s only bullet!</span>")
+		return
+	if(target) /// Misdirection (Scholar of the Wilbur Sin exclusive)
+		user.visible_message("<span class='danger'>[user.name] looks around themselves.</span>",\
+		 "<span class='userdanger'>You look for something to point [src] at.</span>")
+		playsound(loc, 'fulp_modules/lisa/Sounds/misdirect.ogg', 50, FALSE, -5)
+		used_ability = TRUE
+		addtimer(CALLBACK(src, .proc/clear_cooldown), 7 SECONDS)
+		if(do_after(user, 0.4 SECONDS, target = src, progress = FALSE))
+			user.setDir(turn(user.dir, 90))
+		if(do_after(user, 0.3 SECONDS, target = src, progress = FALSE))
+			user.setDir(turn(user.dir, -90))
+		if(do_after(user, 0.3 SECONDS, target = src, progress = FALSE))
+			user.setDir(turn(user.dir, 90))
+		if(do_after(user, 0.3 SECONDS, target = src, progress = FALSE))
+			user.setDir(turn(user.dir, -90))
+		if(do_after(user, 0.3 SECONDS, target = src, progress = FALSE))
+			user.emote("gasp")
+			user.visible_message("<span class='danger'>[user.name] quickly points their gun towards [target]!</span>",\
+			 "<span class='userdanger'>You misdirect [src] towards [target]!</span>",\
+			 "<span class='hear'>You hear a gasp...</span>")
+			for(var/mob/living/H in view(1, target))
+				if(!(H == user) && !(H.stat))
+					if(!H.mind || !istype(H.mind.martial_art, /datum/martial_art/velvetfu))
+						user.visible_message("<span class='danger'>You quickly jump towards [target]!</span>",\
+						 "<span class='userdanger'>[H] quickly jumps towards [target]!</span>",\
+						 "<span class='hear'>You hear aggressive shuffling!</span>")
+						H.Move(target)
+			for(var/mob/living/L in viewers(5, target))
+				L.face_atom(target)
+				L.do_alert_animation()
+		return
+
+/obj/item/gun/ballistic/revolver/joel/process_fire(atom/target, mob/living/user, message = TRUE, params = null, zone_override = "", bonus_spread = 0)
+	add_fingerprint(user)
+	playsound(loc, 'fulp_modules/lisa/Sounds/gunclick.ogg', 50, FALSE, -5)
+	to_chat(user, "<span class='warning'>You don't want to waste [src]'s only bullet!</span>")
+
+
+/*
+ *	Cylinder, bullet & casing.
+ *
+ *	We're using unique ones so people cant get more bullets for anything, yada yada, whatever.
+ *	Unique gun, unique single bullet. You lose it, it's gone.
+ */
+
+/// Cylinder
+/obj/item/ammo_box/magazine/internal/cylinder/c22
+	name = "\improper Bolt Action revolver cylinder"
+	ammo_type = /obj/item/ammo_casing/c22
+	caliber = CALIBER_C22
+	max_ammo = 1
+	multiload = FALSE
+
+/// Bullet casing & Bullet
+/obj/item/ammo_casing/c22
+	name = "5.6mm bullet casing"
+	desc = "An incredibly rare type of bullet."
+	caliber = CALIBER_C22
+	projectile_type = /obj/projectile/bullet/c22
+
+/// This isnt meant to be fired, so do 0 damage. This can technically be fired by dual wielding, so they'll have a nice surprise of a big load of nothing if they do.
+/obj/projectile/bullet/c22
+	name = "5.6mm bullet"
+	damage = 0
```

Comment:
it's still kinda weird to me that this does no damage, even if an admin fires it or something.